docker/base-python/Dockerfile

```diff
@@ -75,7 +75,8 @@ RUN apk --no-cache add \
 # https://github.com/jazzband/pip-tools/#versions-and-compatibility to
 # select a pip-tools version that corresponds to the 'py3-pip' and
 # 'python3' versions above.
-RUN pip3 install pip-tools==6.12.1
+# Pinning build version due to missing license info from pip show in newer versions
+RUN pip3 install pip-tools==6.12.1 build==0.9.0
```

Comment:
To give more context behind this specifically. `pip-tools` does not pin `build` so we're pinning the `build` package to ensure consistency. If you update `pip-tools` you should also update `build` as well. We could have also just updated `py-mkopensource` for the `build` license info but we could run into the same problem again when a new version of `build` is released. So now we deal with any license omissions when we are explicitly updating the version and not randomly.